_sections/30-bindings.md

```diff
@@ -113,37 +113,33 @@ The functions are categorized based on their main usage. There are no rules asso
-<a id="fn:selected-at" href="#fn:selected-at">`selected-at(string list, number index)`</a> | string | Returns the value of the item at the 0-based index of a space-separated list or empty string if the item does not exist (including for negative index and index 0).
-<a id="fn:count-selected" href="#fn:count-selected">`count-selected(node node)`</a> |  number | Returns the number of items in a space-separated list (e.g. `select` data type values).
-<a id="fn:jr:choice-name" href="#fn:jr:choice-name">`jr:choice-name(node node, string value)`</a> |  string | Returns the label value in the active language corresponding to the choice option with the given value of a select or select1 question for the given data node. (sorry)
-<a id="fn:jr:itext" href="#fn:jr:itext">`jr:itext(string arg)`</a> | string | Obtains an itext value for the provided reference in the active language.
-<a id="fn:indexed-repeat" href="#fn:indexed-repeat">`indexed-repeat(node-set arg, node-set repeat1, number index1, [node-set repeatN, number indexN]{0,2})`</a> | string | Returns a single node value from a node-set by selecting the 1-based index of a repeat node-set that this node is a child of. It does this up to 3 repeat levels deep. E.g. `indexed-repeat(//node, /path/to/repeat, //index1, /path/to/repeat/nested-repeat, //index2)` is meant to be a shortcut for `//repeat[position()=//index1]/nested-repeat[position()=index2]/node` in native XPath syntax.
+<a id="fn:join" href="#fn:join">`join(string separator, node-set nodes*)`</a> |  string | Joins the provided arguments using the provide separator between values.
+<a id="fn:substr" href="#fn:substr">`substr(string value, number start, number end?)`</a> |  string | Returns the substring beginning at the specified _0-based_ start index and extends to the character at end index - 1.
+<a id="fn:string-length" href="#fn:string-length">`string-length(string arg)`</a> |  number | Deviates from [XPath 1.0](https://www.w3.org/TR/1999/REC-xpath-19991116/#function-string-length) in that the argument is _required_.
+<a id="fn:contains" href="#fn:contains">`contains(string haystack, string needle)`</a> |  boolean | As in [XPath 1.0](https://www.w3.org/TR/1999/REC-xpath-19991116/#function-contains).
+<a id="fn:starts-with" href="#fn:starts-with">`starts-with(string haystack, string needle)`</a> | boolean | As in [XPath 1.0](https://www.w3.org/TR/1999/REC-xpath-19991116/#function-starts-with).
+<a id="fn:ends-with" href="#fn:ends-with">`ends-with(string haystack, string needle)`</a> | boolean | As in [XPath 3.0](https://www.w3.org/TR/xpath-functions-30/#func-ends-with).
+<a id="fn:uuid" href="#fn:uuid">`uuid(number?)`</a> |  string | Without arguments, it returns a random [RFC 4122 version 4](http://tools.ietf.org/html/rfc4122) compliant UUID. With an argument it returns a random GUID with the provided number of characters.
+<a id="fn:property" href="#fn:property">`property(string prop)`</a> |  string | Tbd
```

Comment:
If used, it's used by Collect users only because it's not supported in Enketo. I never get questions about this though (except Survey123 users - who are using a workaround).

Reply:
Do you know what its intended use was? I don't believe it's used by Collect users (or supported) but perhaps once I know what it's for that will ring a bell.

Reply:
It's used to get username, imei, email etc I think.

Reply:
https://github.com/opendatakit/xforms-spec/issues/57

Reply:
Ah, ok. Well, I still think it can be removed as we discussed in #57. :)